### DISCUSSION TITLE
Can’t launch Swiss on the Nintendo GameCube

### DISCUSSION THREAD

**Discussion by Statebog50** [emukidid/swiss-gc#834] [Q&A]
When I download the latest version of Swiss and burn it to a mini DVD and play it on my Nintendo Gamecube it can’t read the mini DVD. Can you tell me if I am doing anything wrong?

**Comment by emukidid** [comment]
A stock GameCube isn't going to read that disc, you need a way of making it read it, or an alternative way of booting Swiss. Look here for more answers: [gc-forever wiki:Booting_homebrew](https://www.gc-forever.com/wiki/index.php?title=Booting_homebrew)

**Comment by Statebog50** [comment]
Okay. I think I am going to wait and save up for the GameBoy Player Startup
Disk. I looked into the link you sent me and the options. Thank you.

**Comment by emukidid** [comment]
No problem. The cost of that disc is the same as buying another modded GameCube console.

**Comment by Statebog50** [comment]
I just received my Game Boy Player Startup Disc today after ordering one
from eBay and choosing to pay for it on 4 bi weekly purchases through
Klarna and it works great with my Game Boy Player Accessory connected to my
Nintendo GameCube Console.
